bbp/modules/default.nix

```diff
@@ -238,11 +238,22 @@ let
             name = "pytouchreader";
             moduleFilePrefix = "nix/hpc";
             description = "pytouchreader module generated by nix";
-            packages =
-            [
+            packages = with pkgs.pythonPackages; [
                 pkgs.pytouchreader
-            ] ++ pkgs.pytouchreader.pythonDeps;
-            conflicts = conflicts-modules;
+            ] ++ ( getPyModRec [ pkgs.pytouchreader ] ++ [ add-site-dir ] );
```

Comment:
There is no need to provide `pkgs.pytouchreader` manually because `getPyModRec` already provides it. (see `bb5` attribute definition in this file).

Reply:
Thanks, fixed!